.github/workflows/linting.yml

```diff
@@ -124,6 +132,11 @@ jobs:
               - '**/eslint.config.*'
               # If the excludelist changed, run to ensure newly non-excluded files pass.
               - 'tools/eslint-excludelist.json'
+            misc_css:
+              # If Stylelint config changed, there may be new checks.
+              - .stylelintignore
+              - stylelint.config.mjs
```

Comment:
Is it possible to have a stylelint config in a subdir, like eslint does? If so, adding something like this might be a good idea.
```suggestion
              - stylelint.config.mjs
              - '**/stylelint.config.{js,mjs,cjs}'
```

Reply:
It only accepts a single config file, and [searches from the current working dir upward](https://stylelint.io/user-guide/configure/), so effectively only the one the current working dir is relevant.

Reply:
Ah, indeed. In fact I tested and it does seem to be relative to the current file, so I'll use your suggestion.